src/main/kotlin/fi/metatavu/metaform/server/controllers/AuditLogEntryController.kt

```diff
@@ -18,6 +19,10 @@ class AuditLogEntryController {
     @Inject
     lateinit var auditLogEntryDAO: AuditLogEntryDAO
 
+    @Inject
+    @ConfigProperty(name = "metaforms.features.auditlog")
+    lateinit var auditLog: String
```

Comment:
This should probably be boolean, not string. It also should default to true, not false. Otherwise it will break old installations

Reply:
Resolved